input/fsh/vaccination/cs-napkon-vaccination.fsh

```diff
@@ -7,81 +7,14 @@ Description: "NAPKON Vaccination Module Code System - Collection of vaccination
-* #2003 "Um mein Arbetsfeld vor einer Infektion mit SARS-CoV-2 bzw. COVID-19 zu schützen"
-* #2004 "Weil alle sich impfen lassen"
-* #2005 "Weil ich befürchte aufgrund meines Impfstatus' benachteiligt zu werden (z.B. Reiseverbot, Arbeitsverbot)"
-
-// vaccination injection site pain scale
-* #3001 "Leichte Schmerzen, diese stören nicht bei täglichen Aktivitäten"
-* #3002 "Mäßige Schmerzen, diese stören bei täglichen Aktivitäten"
-* #3003 "Sehr starke Schmerzen, die dazu führen, dass täglichen Aktivitäten nicht mehr durchgeführt werden können"
-
-// vaccination injection site erythema/swelling size
-* #4001 "Kleiner als 2,5 Zentimeter"
-* #4002 "Zwischen 2,5 und 5 Zentimeter"
-* #4003 "Zwischen 5,1 und 10 Zentimeter"
-* #4004 "Größer als 10 Zentimeter"
-
-// medical treatment to potential or actual adverse reactions to vaccination
-* #6001 "Ja, Behandlung in Arztpraxis oder Rettungsstelle"
-* #6002 "Ja, Behandlung mit Aufnahme im Krankenhaus"
-
-// body temperature categories
-* #7001 "Weniger als 37,5 °C"
-* #7002 "37,5 °C bis 37,9 °C"
-* #7003 "38,0 °C bis 38,4 °C"
-* #7004 "38,5 °C bis 38,9 °C"
-* #7005 "39,0 °C bis 40,0 °C"
-* #7006 "Mehr als 40 °C"
-
-// alcoholic beverage intake
-* #8001 "Einmal im Monat oder seltener"
-* #8002 "2-4 Mal im Monat"
-* #8003 "2-3 Mal pro Woche"
-* #8004 "4 Mal oder öfter die Woche"
-* #8005 "Nicht mehr (früher gelegentlich)"
-
-// alcoholic beverage intake per day
-* #9001 "0,5 bis 1 alkoholische Getränke pro Tag"
-* #9002 "1,5 bis 2 alkoholische Getränke pro Tag"
-* #9003 "2,5 bis 3 alkoholische Getränke pro Tag"
-* #9004 "3,5 bis 4,5 alkoholische Getränke pro Tag"
-* #9005 "5 oder mehr alkoholische Getränke pro Tag"
-
-// allgemeine Beschwerden nach Impfung
-* #10001 "Fieber und/oder Schüttelfrost (inkl. sich fiebrig fühlen)"
-* #10002 "Abgeschlagenheit und/oder vermehrte Müdigkeit"
-* #10003 "Neu aufgetretene oder Verstärkung vorbestehender Muskelschmerzen"
-* #10004 "Neu aufgetretene oder Verstärkung vorbestehender Gelenkschmerzen"
-* #10005 "Schwierigkeiten beim Atmen und/oder Luftnot"
-* #10006 "Übelkeit und/oder Erbrechen"
-* #10007 "Durchfall und/oder dünnerer Stuhl"
-* #10008 "Abgeschlagenheit und/oder vermehrte Müdigkeit"
-
-// complaint intensity
-* #11001 "Ein wenig (keine Einschränkung täglicher Aktivitäten)"
-* #11002 "Mäßig bis stark (Einschränkung täglicher Aktivitäten)"
-* #11003 "Sehr stark (tägliche Aktivitäten sind nicht mehr möglich)"
-* #11004 "Potenziell lebensbedrohlich (Behandlung in Rettungsstelle oder Hospitalisierung)"
+* #2001 "To protect myself from infection with SARS-CoV-2 or from COVID-19"
+* #2002 "To protect my private environment from infection with SARS-CoV-2 or COVID-19"
+* #2003 "To protect my work environment from infection with SARS-CoV-2 or COVID-19"
+* #2004 "Because everyone gets vaccinated"
+* #2005 "Because I am afraid of being discriminated against due to my vaccination status (e.g. travel ban, work ban)."
+
+* #no-longer "No longer"
```

Comment:
This depends on the use case. In case you'd like to be able to differentiate between patients that generally do not drink alcohol and patients that no longer do, yes we should keep it like that. However, as of now, there is no additional data captured as to how much alcohol a patient has consumed before. This results in 'No longer' being fairly vague, one patient might have consumed alcohol daily in large quantities before quitting it altogether, one patient might have had a glass of wine every other week. Without this additional information, the difference between "No longer" and "0" therefore becomes almost irrelevant. 

This is my thought process, feel free to make your own call based on that information 👍

Reply:
I agree regarding the vagueness of the information, but given that the physicians have specified this answer option they apparently consider it a valuable information. I'll therefore leave it as an option for now.